hieradata/node/auxtel-fp01.cp.lsst.org.yaml

```diff
@@ -51,7 +91,7 @@ nfs::client_enabled: true
 nfs::client_mounts:
   /net/self/data:
     share: "data"
-    server: "%{facts.networking.fqdn}"
+    server: "auxtel-fp01.cp.lsst.org"
```

Comment:
Why does this need to be changed?

Reply:
No particular reason, i have updated the code.